src/commons/components/Input/Input.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styles from './Input.module.css';
+import cn from 'classnames';
+
+export interface InputProps {
+    size?: 's' | 'm' | 'l';
+    iconLeft?: React.ReactNode;
+    className: string;
+    containerClassName?: string;
+}
+
+export const Input: React.FC<InputProps & React.DetailedHTMLProps<React.InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>> = ({
```

Comment:
Could you please move this type in generic to something like `type PropsWithHTMLInput`?